app.js

```diff
@@ -0,0 +1,94 @@
+totalCountry.textContent = `The toal number of countries are: ${myCountries.length}`; //give h5 content
+
+const parentDiv = document.getElementById('parentDiv'); //selected the parent element with class "row"
+// Function to display country
+function displayCountries(displayEachCountry) {//Take the countires as a parameter
+    
+    parentDiv.innerHTML = "";//clear the parent div
+    //display new countries it receives
+    displayEachCountry.forEach((country) => {
+        const firstDiv = document.createElement('div'); // create the first div
+        firstDiv.classList = "col-sm-3"; //first div class of col-sm-2
+        const secondDiv = document.createElement('div'); //created the second div
+        secondDiv.className = "card"; //second div class of card
+        secondDiv.classList.add("mb-4") //second div with margin buttom 
+        const mainDiv = document.createElement('div'); //created the main div to house the content of the array
+        mainDiv.className = "card-body"; //class of card-body
+        mainDiv.classList.add("p-5") //class of padding
+        mainDiv.classList.add("text-center") //class of text center
+        mainDiv.classList.add('text-light') //text color white
+        mainDiv.textContent = country; //main div content
+
+        parentDiv.appendChild(firstDiv); //append the first div to the parent div
+        firstDiv.appendChild(secondDiv); //append the second div to the first div
+        secondDiv.appendChild(mainDiv); //append the main div to the second div 
+    })
+}
+displayCountries(myCountries); //called the function to display country
+
+//The Form Element
+const firstButton = document.forms['search'].querySelector('#starting-word'); //i selected the first button
+const secondButton = document.forms['search'].querySelector('#any-word')// i selected the second button
+
+const numberOfCountries = document.getElementById("display"); //i selected the p element with id display
+const listOfCountries = []; //i created an empty array to put countries to be displayed
+
+firstButton.addEventListener('click', function(e) { //add an event to the first button
+    e.preventDefault(); //prevents the page from refreshing
+
+    const value = document.forms['search'].querySelector('.input').value.toLowerCase(); //get the value of input
+    const countries = document.querySelectorAll('div .card-body'); //get the div element housing the countries
+    Array.from(countries).forEach((country)=> { //loop through the array of html collections
+        let eachCountry = country.textContent //text content of each html collection, i put inside a variable of eachCountry
+        if (eachCountry.toLowerCase().startsWith(value)) { //if the text content start with the value inputed
+            listOfCountries.push(eachCountry) //push it inside an empty array of name listOfCountries
+            displayCountries(listOfCountries); // call the function display countires and give it an argument 
```

Comment:
This should be called only once, i.e after you have finished with filtering the countries